specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/examples/DiskMigrations/Cancel.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "04666444-56f0-4d4f-afc5-dbd6b134b084",
+    "location": "redmond",
+    "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+    "api-version": "2018-07-30-preview"
+  },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/04666444-56f0-4d4f-afc5-dbd6b134b084/providers/Microsoft.Compute.Admin/locations/redmond/diskmigrations/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+                "name": "redmond/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+                "type": "Microsoft.Compute.Admin/locations/diskmigrations",
+                "location": "redmond",
+              "properties": {
+                "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+                "status": "Canceled",
```

Comment:
This is an anti-pattern since diskmigration is an operation and not a resource. Typically, "canceled" is a bad status to have, since it is an operation name. Resource status should be something that describes is's state - OK, Failed, etc.

Reply:
In this resource design, the resource is job. And a job could be in a state - canceled.